editor/shader/shader_editor_plugin.cpp

```diff
@@ -868,92 +849,75 @@ ShaderEditorPlugin::ShaderEditorPlugin() {
+	shader_dock->add_child(files_split);
 
 	context_menu = memnew(PopupMenu);
-	add_child(context_menu);
 	context_menu->connect(SceneStringName(id_pressed), callable_mp(this, &ShaderEditorPlugin::_menu_item_pressed));
-
-	make_floating = memnew(ScreenSelect);
-	make_floating->connect("request_open_in_screen", callable_mp(window_wrapper, &WindowWrapper::enable_window_on_screen).bind(true));
-	if (!make_floating->is_disabled()) {
-		// Override default ScreenSelect tooltip if multi-window support is available.
-		make_floating->set_tooltip_text(TTR("Make the shader editor floating.") + "\n" + TTR("Right-click to open the screen selector."));
-	}
-
-	window_wrapper->connect("window_visibility_changed", callable_mp(this, &ShaderEditorPlugin::_window_changed));
+	add_child(context_menu);
 
 	shader_list = memnew(ItemList);
 	shader_list->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
 	shader_list->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	shader_list->set_theme_type_variation("ItemListSecondary");
 	shader_list->set_custom_minimum_size(Size2(100, 60) * EDSCALE);
-	files_split->add_child(shader_list);
 	shader_list->connect(SceneStringName(item_selected), callable_mp(this, &ShaderEditorPlugin::_shader_selected).bind(true));
 	shader_list->connect("item_clicked", callable_mp(this, &ShaderEditorPlugin::_shader_list_clicked));
 	shader_list->set_allow_rmb_select(true);
 	SET_DRAG_FORWARDING_GCD(shader_list, ShaderEditorPlugin);
-
-	main_container->add_child(files_split);
-	main_container->set_custom_minimum_size(Size2(100, 300) * EDSCALE);
-
-	VBoxContainer *shader_vb = memnew(VBoxContainer);
-	shader_vb->set_h_size_flags(Control::SIZE_EXPAND_FILL);
-	files_split->add_child(shader_vb);
-	empty_menu = memnew(HBoxContainer);
-	shader_vb->add_child(empty_menu);
-	empty_menu->add_child(file_menu);
-	empty_menu->add_theme_style_override(SceneStringName(panel), EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("ScriptEditorPanel"), EditorStringName(EditorStyles)));
+	files_split->add_child(shader_list);
 
 	shader_tabs = memnew(TabContainer);
 	shader_tabs->set_custom_minimum_size(Size2(460, 300) * EDSCALE);
 	shader_tabs->set_tabs_visible(false);
 	shader_tabs->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 	shader_tabs->set_v_size_flags(Control::SIZE_EXPAND_FILL);
-	shader_vb->add_child(shader_tabs);
-	Ref<StyleBoxEmpty> empty;
-	empty.instantiate();
+	Ref<StyleBoxEmpty> empty(memnew(StyleBoxEmpty));
 	shader_tabs->add_theme_style_override(SceneStringName(panel), empty);
+	shader_tabs->hide();
+	files_split->add_child(shader_tabs);
 
-	EditorNode::get_bottom_panel()->add_item(TTRC("Shader Editor"), window_wrapper, ED_SHORTCUT_AND_COMMAND("bottom_panels/toggle_shader_editor_bottom_panel", TTRC("Toggle Shader Editor Bottom Panel"), KeyModifierMask::ALT | Key::S));
+	file_menu = memnew(MenuButton);
+	file_menu->set_flat(false);
+	file_menu->set_theme_type_variation("FlatMenuButton");
+	file_menu->set_text(TTR("File"));
```

Comment:
```suggestion
	file_menu->set_text(TTRC("File"));
```
If you are changing it anyway.